install/kubernetes/cilium/values.yaml

```diff
@@ -9,6 +9,8 @@ agent:
   keepDeprecatedLabels: false
   # Keep the deprecated probes when deploying Cilium DaemonSet
   keepDeprecatedProbes: false
+  # Additional annotations for the agent pods
+  podsAnnotations:
```

Comment:
I think these should maybe be introduced in `install/kubernetes/cilium/charts/{agent,operator}/values.yaml` instead?

Reply:
absolutely, thanks @joestringer, I moved them!